app/controllers/medicaid/amounts_expenses_controller.rb

```diff
@@ -1,11 +1,47 @@
 module Medicaid
-  class AmountsExpensesController < MedicaidStepsController
+  class AmountsExpensesController < Medicaid::MemberStepsController
+    def current_member
+      @_current_member ||= super || first_member_with_expenses
+    end
+
+    def update
```

Comment:
one thing @disambiguator and I had been starting to try was using the `after_successful_update_hook` and defining model-specific params from the step_params (e.g. `member_params` that could just be `step_params.merge(member_id: current_member.id)`). the idea behind that is that it exercises our existing interface, surfaces pain points, and allows us to have a contained site for change in future. may be worth trying here?

Reply:
I like the idea of `member_params`. One slight suggestion I have is to give it one nudge towards more specificity and call it `member_step_params`. That'd keep the `step` context in line.

Reply:
@jayroh Dig it.

We should move in that direction, although on second look this one deviates just ever so slightly from that pattern and we can't just do the quick refactor right now. Gonna save this one for later.